app/views/products/_measure_selection.html.erb

```diff
@@ -1,3 +1,12 @@
+<%
+
+# local variables
+#
+#   f   (product form)

```

Comment:
This partial gets re-rendered by ajax if the user changes the bundle during product creation. (see assets/javascripts/measure_selection.js : UpdateMeasureSet --> measures_controller.grouped) 

Right now this is breaking because we don't have a form object when re-rendering. We could possibly work around that but I think it would be easier to just avoid using a form object in this partial. The new version is definitely cleaner though.